src/app/client/src/app/modules/learn/learn-routing.module.ts

```diff
@@ -19,6 +20,13 @@ const routes: Routes = [
       telemetry: { env: telemetryEnv, pageid: 'learn', type: 'view' }
     }
   },
+  {
+    path: 'redirect', component: RedirectComponent,
+    data: {
+      breadcrumbs: [{ label: 'Home', url: '/home' }, { label: 'Redirect', url: '' }],
```

Comment:
why is there a breadcrumb for this and why is it having home mentioned?

Reply:
All the pages have 'Home' by default in the breadcrumb, So i added, if not required, l can remove it.

Reply:
please remove it as we are not on the same page where you can give a breadcrumb we have opened new tab